f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -106,7 +106,7 @@ def _update_payload(self, old_resource, resource, service, **kwargs):
     def _create(self, bigip, payload, resource, service, **kwargs):
         resource_helper = kwargs.get("helper", self.resource_helper)
         resource_type = kwargs.get("type", self._resource)
-        overwrite = kwargs.get("overwrite", True)
+        overwrite = kwargs.get("overwrite", False)
```

Comment:
If only vip is not mutable, you may handle it in vip's _create(), instead of its base class.

Reply:
Yes, I just push another commit again.